### PR TITLE
Updating extensions README

### PR DESCRIPTION
**Pull request by Meggin**
Fixes this issue: https://github.com/ampproject/amphtml/issues/1741

### CODE REVIEW

**Comment by cramforce** [comment]
Ping @dgash for CLA

**Comment by Meggin** [comment]
Will do.

File: extensions/amp-access/amp-access-spec.md [hunk 1]
```diff
@@ -118,18 +117,18 @@ Here’s an example of the AMP Access configuration:
...
-RANDOM            | A random number. Helpful to avoid browser cache.
+| Var               | Description |
+| ----------------- | ----------- |
+| READER_ID         | The AMP Reader ID. |
+| AUTHDATA(field)   | The value of the field in the authorization response. |
+| RETURN_URL        | The placeholder for the return URL specified by the AMP runtime for a Login Dialog to return to. |
+| AMPDOC_URL        | The URL of this AMP Document. |
+| CANONICAL_URL     | The canonical URL of this AMP Document. |
+| DOCUMENT_REFERRER | The Referrer URL. |
+| VIEWER            | The URL of the AMP Viewer. |
```
Comment (cramforce) [inline extensions/amp-access/amp-access-spec.md]: I don't think this exists.

File: extensions/amp-access/amp-access-spec.md [hunk 2]
```diff
@@ -118,18 +117,18 @@ Here’s an example of the AMP Access configuration:
 
 ##Access URL Variables
 
-When configuring the URLs for various endpoints, the Publisher can use substitution variables. These variables are a subset of the variables defined in the [AMP Var Spec](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md). The set of allowed variables is defined in the table below:
+When configuring the URLs for various endpoints, the Publisher can use substitution variables. These variables are a subset of the variables defined in the [AMP Var Spec](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md). The set of allowed variables is defined below.
```
Comment (cramforce) [inline extensions/amp-access/amp-access-spec.md]: Actually https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md has the full list and the table is a mix of relevant fields and some additional substitutions

File: extensions/amp-access/amp-access-spec.md [hunk 3]
```diff
@@ -140,28 +139,28 @@ https://pub.com/access?
   &_={RANDOM}
 ```
 
-AUTHDATA variable is availbale to Pingback and Login URLs. It allows passing any field in the authorization
-response as an URL parameter. E.g. `AUTHDATA(isSubscriber)`.
+The AUTHDATA variable is availbale to Pingback and Login URLs. It allows passing any field in the authorization
```
Comment (cramforce) [inline extensions/amp-access/amp-access-spec.md]: available.

**Comment by cramforce** [comment]
Wow, this is tough to review.
+ @rudygalfi @sebastianbenz

**Comment by Meggin** [comment]
I will take out the copy edits from this PR and create a separate one, and make sure this is a clean PR. 
I honestly didn't realize when I merged Dave's copy edit commits in my fork that they would automatically get pulled into this PR.

**Pull request closed without merging**